synapse/lib/storm.py

```diff
@@ -713,19 +713,20 @@ def _reqOperArg(self, oper, name):
 
     def _cmprCtorHas(self, oper):
         prop = self._reqOperArg(oper, 'prop')
+
         def cmpr(tufo):
             return tufo[1].get(prop) is not None
 
         return cmpr
 
     def _cmprCtorIn(self, oper):
-        prop = self._reqOperArg(oper, 'prop')
-        valus = self._reqOperArg(oper, 'valu')
-        if len(valus) > 12: # TODO opt value?
-            valus = set(valus)
+        prop, valu = oper[1].get('args')
```

Comment:
Like the `range` filter, it was not possible to get `prop` and `valus` into this fn.